### ISSUE TITLE
SP API App Draft status Vs Published Status

### ISSUE THREAD

**Issue by DasariArchana** [amzn/selling-partner-api-docs#2064]
Hi 

We followed the documentation for 'Website authorization workflow', our OAuth Authorization URI has the flag version=beta as our app is in draft state and we are successfully able to make SP API calls. But we wanted to understand if there is any significant difference between an SP API app in DRAFT status Vs PUBLISHED status ?

Can we keep the app in a draft state without publishing it?

@rugved1991 can you please throw some light on this ?

**Comment by rugved1991** [comment]
Hi @DasariArchana,

You can keep the app in draft state. The only change when the app is published is that it is made available for all sellers to use on the AppStore. Also, you will be able to migrate your MWS authorizations to SP API only when the app is in published status.

Rugved
Solutions Architect, SP API

**Comment by grmiatov** [comment]
@rugved1991 

Hello.

We have also created a new (SP API only) app in draft state and started testing it with sellers authorising SP API access in a number of regions. I would like to know what happens to SP API authorizations we have obtained from sellers for the draft app when the app is published.

Do they remain valid or do sellers have to re-authorize the app once it is published? If they remain valid, is there any limit or expiry date for the authorisations?

Thank you.
